helm/external-dns-app/values.schema.json

```diff
@@ -12,19 +12,19 @@
                     "type": "string"
                 },
                 "baseDomain": {
-                    "type": ["null", "string"]
+                    "type": "null"
```

Comment:
careful with these (https://intranet.giantswarm.io/docs/organizational-structure/teams/cabbage/app-updates/helm-values-schema/#caveats)

Reply:
I've fixed all of them now. Please re-review